packages/app/src/vis-packs/nexus/hooks.ts

```diff
@@ -63,14 +68,19 @@ export function useAxisMapping(
 
 export function useAuxiliaries(
   auxDatasets: NumArrayDataset[],
-  selection: string | undefined
+  selection: string | undefined,
+  auxErrorsDatasets: NumArrayDataset[] = []
 ): Auxiliary[] {
   const { attrValuesStore } = useDataContext();
 
-  const auxValues = useDatasetValues(auxDatasets, selection);
+  const auxValues = useDatasetValues(
+    [...auxDatasets, ...auxErrorsDatasets],
+    selection
+  );
 
   return auxDatasets.map((dataset) => ({
     label: getDatasetLabel(dataset, attrValuesStore),
     value: auxValues[dataset.name],
+    error: auxValues[`${dataset.name}_errors`],
```

Comment:
`auxErrorsDatasets` may not have the same number of entries than `auxDatasets` (one could imagine having aux datasets with errors and some without) so I can't rely on the index in the array.

I instead reconstruct the key in `auxValues` from the dataset name. Again, not great. Perhaps we could find a way to abstract this reconstruction to be used in `findErrorsDataset` ?

Reply:
Ooh smart, taking a look at it now.

Reply:
Much better indeed :clap: